### DISCUSSION TITLE
Annotate gene activity matrix with SCT normalized scRNA data?

### DISCUSSION THREAD

**Discussion by danielcgingerich** [stuart-lab/signac#510] [Q&A] [answered]
Hi, I have an integrated reference from 14 datasets.  Each dataset was SCT normalized prior to integration.  This reference was used to annotate our experimental scRNA data.  We also need to annotate scATAC data.  Can I use the SCT normalized reference?  Or should I make another reference from the same data that is normalized with the default method before annotating the gene activity matrix?

Thanks,
Dan

**Comment by timoast** [comment, marked as answer]
I would recommend using the log-normalised RNA data for the reference. In general, you want to use the same normalisation method for reference and query, and since you can't SCTransform the gene activity assay, you should log-normalise the reference scRNA-seq data.
